src/nvim/drawscreen.c

```diff
@@ -2643,7 +2643,6 @@ int number_width(win_T *wp)
 /// Set must_redraw only if not already set to a higher value.
 /// e.g. if must_redraw is UPD_CLEAR, type UPD_NOT_VALID will do nothing.
 void redraw_later(win_T *wp, int type)
-  FUNC_ATTR_NONNULL_ALL
```

Comment:
was this intentionally removed? `wp` must not be null.

Reply:
When exiting `curwin` may be null, and it is not accessed because `exiting` is `true`. It may be replaced by an assertion, but accessing a null pointer will segfault anyway, so an assertion isn't really different.

Reply:
Ok. An assertion may avoid accidental reintroduction of FUNC_ATTR_NONNULL_ALL here in the future.